grants/casper_stats/data.md

```diff
@@ -0,0 +1,60 @@
+---
```

Comment:
Casper Stats milestone details contain details about Ramp Network. Could you please check and update as you see fit?